.github/workflows/ci.yml

```diff
@@ -21,17 +21,26 @@ jobs:
       - uses: actions/setup-python@v5
       - name: Install dependencies
         run: |
+          python -m venv venv
+          source venv/bin/activate
+          python -m pip install --upgrade pip
```

Comment:
why?

Reply:
The > python: /lib/x86_64-linux-gnu/libm.so.6: version `GLIBC_2.38' not found  
is not needed. Will remove